<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoMainWindow</class>
 <widget class="QMainWindow" name="TodoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TodoMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>添加</string>
    </property>
    <addaction name="action_addNewDate"/>
    <addaction name="action_addNewField"/>
    <addaction name="action_addNewMonthy"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>打开日志</string>
    </property>
    <addaction name="action_open_logger"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_addNewDate">
   <property name="text">
    <string>添加新的日期计划</string>
   </property>
  </action>
  <action name="action_addNewField">
   <property name="text">
    <string>添加新的领域计划</string>
   </property>
  </action>
  <action name="action_addNewMonthy">
   <property name="text">
    <string>添加新的月份计划</string>
   </property>
  </action>
  <action name="action_open_logger">
   <property name="text">
    <string>打开日志</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
